frontend/src/views/OfferView.vue

```diff
@@ -29,13 +33,28 @@ let offerInfo = ref<OfferCreate | Offer>({
   items: [],
 });
 
+let offerItemsArray = ref<any>([]);
+
+function addOfferItem(type: ArticleKind) {
+  offerItemsArray.value.push({ id: itemCount++, type: type });
```

Comment:
It must be an id because it will be used as temporary id for new list elements. If there are an existing positions we will use an id which came from backend.